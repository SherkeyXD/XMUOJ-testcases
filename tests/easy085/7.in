-12 32
0 25
-26 40
18 18
-14 26
10 27
-35 0
9 34
-12 27
10 34
-12 6
3 16
-13 44
15 30
-10 30
18 42
-33 20
10 17
-8 15
24 33
-9 19
33 37
0 0