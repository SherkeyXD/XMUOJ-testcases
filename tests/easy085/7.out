OK
OK
Sorry
OK
OK
OK
OK
OK
OK
OK
OK
OK
Sorry
OK
OK
Sorry
OK
OK
OK
OK
OK
OK
